Formula/p/pango.rb

```diff
@@ -6,6 +6,14 @@ class Pango < Formula
   license "LGPL-2.0-or-later"
   head "https://gitlab.gnome.org/GNOME/pango.git", branch: "main"
 
+  # Pango doesn't follow GNOME's "even-numbered minor is stable" version
+  # scheme but they do appear to use 90+ minor/patch versions, which may
+  # indicate unstable versions (e.g., 1.99.0, 2.2.99.0, etc.).
+  livecheck do
+    url "https://download.gnome.org/sources/pango/cache.json"
+    regex(/pango-(\d+(?:(?!\.9\d)\.\d+)+)\.t/i)
```

Comment:
```suggestion
  # indicate unstable versions (e.g., 1.90, etc.).
  livecheck do
    url "https://download.gnome.org/sources/pango/cache.json"
    regex(/pango[._-]v?(\d+(?:(?!\.9\d)\.\d+)+)\.t/i)
```

The versions in the comment are specific to `tracker`, so this suggestion updates it to use an example from `pango`. This also tweaks the regex to bring it more in line with typical patterns (though the existing regex was simply a modified version of the generated strategy regex).